draft-packages/form/KaizenDraft/Form/Primitives/InputSearch/InputSearch.tsx

```diff
@@ -0,0 +1,97 @@
+import { Icon } from "@kaizen/component-library"
+import { LoadingSpinner } from "@kaizen/draft-loading-spinner"
+import classnames from "classnames"
+import React from "react"
+import search from "@kaizen/component-library/icons/search.icon.svg"
+import clear from "@kaizen/component-library/icons/clear.icon.svg"
+
+import styles from "../Input/styles.scss"
+
+export interface InputSearchProps
+  extends Omit<
+    React.InputHTMLAttributes<HTMLInputElement>,
+    "className" | "defaultValue"
+  > {
+  classNameAndIHaveSpokenToDST?: string
+  reversed?: boolean
+  loading?: boolean
+  secondary?: boolean
+  onClear?: () => void
+}
+
+const InputSearch: React.FunctionComponent<InputSearchProps> = (
+  props: InputSearchProps
+) => {
+  const {
+    value,
+    onChange,
+    onClear,
+    classNameAndIHaveSpokenToDST,
+    placeholder,
+    disabled = false,
+    reversed = false,
+    loading,
+    secondary,
+    ...genericInputProps
+  } = props
+
+  return (
+    <div
+      className={classnames(
+        styles.wrapper,
+        styles.withStartIconAdornment,
+        styles.withEndIconAdornment,
+        styles.withSearch,
+        {
+          [styles.withReversed]: reversed,
+          [styles.withDisabled]: disabled,
+          [styles.withSecondary]: secondary,
+          [styles.withLoading]: loading,
+        }
+      )}
+    >
+      <div className={styles.startIconAdornment}>
+        {loading ? (
+          <div className={styles.loadingIcon}>
+            <LoadingSpinner accessibilityLabel="Loading comments" size="sm" />
```

Comment:
🤔 What is `comments` in this context? I would probably go for an empty string here because having important information in an icon that precedes the input in the DOM isn't going to be helpful for assistive tech. Not sure exactly how we're going to tackle that issue just yet.